style.css

```diff
@@ -201,10 +201,11 @@ main {
 
 .popup-window {
   position: absolute;
-  width: 1108px;
+  width: 1156px;
   height: 1163px;
   left: 142px;
   top: 77px;
+  border-radius: 16px;
   background-color: #fff;
   z-index: 2;
 }
```

Comment:
- [X] Please fix the below issues for desktop screens and make sure your design matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=JisnhgjX2CTJQVt3-0) design.


  - Please make sure that the project title, tags, and images start on the same vertical line.
  - Please make sure that the project tags and button section are aligned on the right side of the pop-up(model), with some space right away from the wall.
  - Please add the below interactions to the pop-up(model) buttons. 

Button interactivity 👇 

<img src="https://user-images.githubusercontent.com/83803414/208910865-14f15413-14e0-442a-88a5-c65a93401903.png">

Your design 👇 

![hen-pp](https://user-images.githubusercontent.com/78634841/209137333-97c0ece4-7217-4a38-ba70-8e10364175c8.png)
![hen-pp1](https://user-images.githubusercontent.com/78634841/209137364-53cbc842-2611-45b0-a0f0-f89dae25daf9.png)

Figma design 👇 

<img src="https://user-images.githubusercontent.com/83803414/208912815-5c90e2c0-fc39-41cd-a417-937878540ff2.png">
<img src="https://user-images.githubusercontent.com/83803414/208912755-cc1fb260-1bcb-4804-88a2-5268f8416d41.png">